hack/tests/sanity-check.sh

```diff
@@ -10,5 +10,10 @@ go fmt ./...
 ./hack/generate/cli-doc/gen-cli-doc.sh
 go run ./hack/generate/changelog/gen-changelog.go -validate-only
 
+make install
```

Comment:
we are installing the sdk binary to be able to use operator-sdk to gen the samples and check its result via git diff in the sanity to ensure that all are updated with the latest changes.

Reply:
@camilamacedo86 that's what I mean. I usually run `make test-sanity` before submitting my PR. And this will install the SDK locally on my machine without me knowing. Leave it for now, if it annoys me too much I'll find a way to fix it so it doesn't annoy me :)

Reply:
@jmrodri it might make sense here https://github.com/operator-framework/operator-sdk/blob/master/.travis.yml#L110

Though, I think you'd have to run make install to pick up any scaffolding changes locally-- without that, changes to the scaffolding without changes to the sample could erroneously pass.